[docs] Add an announcement for v1.9.0 (#5361)
DevExpress/testcafe: docs/articles/blog/2020-07-27-testcafe-v1-9-0-released.md

    await t
        .click(googleButton)
        .typeText('input[type=email]', 'This text will be entered inside the pop-up');
});
```

You can use the [t.openWindow]() method to open a child window from test code:

```js
import { Selector, ClientFunction } from 'testcafe';

fixture `Test page`
    .page('https://devexpress.github.io/testcafe/example/');

test('Open a new window', async t => {
    await t.openWindow('http://example.com');

    const url = await t.eval(() => document.documentURI);
    await t.expect(url).eql('http://example.com/');
});
```

The [t.switchToWindow]() method enables you to switch between open windows. You can use a window identifier or a predicate to specify the window that should be activated:

```js
fixture `Example page`
    .page('http://example.com');

test('Switch to a specific window', async t => {
    const initialWindow = await t.getCurrentWindow();
    const popUp1 = await t.openWindow('http://example1.com');
    const popUp2 = await t.openWindow('http://example2.com');

    await t.switchToWindow(initialWindow);

    const url = t.eval(() => document.documentURI);
    await t.expect(url).eql('http://example.com/');

    await t

>         .switchToWindow(w => w.location.host === 'example1.com')

Reviewer: we do not use `location` anymore.
Now it's `url`. The description of the url object is here: https://nodejs.org/api/url.html#url_the_whatwg_url_api

```js
fixture `Example page`
    .page('http://www.example.com');

test('Close the current window', async t => {
    const window1 = await t.openWindow('http://devexpress.com');

    await t.closeWindow();

    const url = await t.eval(() => document.documentURI);

    await t.expect(url).eql('http://www.example.com/');
});

test('Close a specific window', async t => {
    const window1 = await t.openWindow('http://devexpress.com');

    await t.closeWindow(window1);
});
```

### Detailed Diffs in Failed Assertions

Test run reports now show the differences between an assertion's actual and expected values:

<img srcset="../images/blog/2020-07-23-rich-diffs.png,
            ../images/blog/2020-07-23-rich-diffs@2x.png 2x"
        src="../images/blog/2020-07-23-rich-diffs.png"
        alt="A report showing differences between asserted values"/>

TestCafe can display difference between values, arrays, objects, and even functions.

## Bug Fixes

* TestCafe now throws a descriptive error when it attempts to start the browser UI on Linux without the X11 server ([4461](https://github.com/DevExpress/testcafe/issues/4461))

> * Fixed an exception thrown when you use remote browsers on Linux without X11 or run Windows browsers from WSL2 ([#4742](https://github.com/DevExpress/testcafe/issues/4742))

Reviewer 2: ```suggestion
* Exception no longer thrown when you use remote browsers on Linux without X11 or run Windows browsers from WSL2 ([#4742](https://github.com/DevExpress/testcafe/issues/4742))
```
